Doc/tutorial/errors.rst

```diff
@@ -101,29 +101,29 @@ The :keyword:`try` statement works as follows.
 * If no exception occurs, the *except clause* is skipped and execution of the
   :keyword:`try` statement is finished.
 
-* If an exception occurs during execution of the try clause, the rest of the
+* If an exception occurs during execution of the *try clause*, the rest of the
   clause is skipped.  Then if its type matches the exception named after the
```

Comment:
This is not part of the PR, but this minor grammar fix should be made anyways. Wouldn't be worth it's own PR.
```suggestion
   clause is skipped.  Then, if its type matches the exception named after the
```